diff -uN leedsatl.orig/satl1.f leedsatl/satl1.f
--- leedsatl.orig/satl1.f	2021-07-13 12:15:06.000000000 +0900
+++ leedsatl/satl1.f	2021-07-13 12:13:31.000000000 +0900
@@ -59,7 +59,7 @@
 C
 C ============================================================================
 C
-      PARAMETER (IPLMAX=8,IPCLM=23000)
+      PARAMETER (IPLMAX=12,IPCLM=81809)
 C
 C ============================================================================
 C
@@ -77,8 +77,10 @@
 C
 C ============================================================================
 C
-      PARAMETER (INLAY=14,INTAU=2,IPCAA=26000)
-      PARAMETER (INST1=5,INST2=2,INST1B=3)
+      ! PARAMETER (INLAY=14,INTAU=2,IPCAA=26000)
+      ! PARAMETER (INST1=5,INST2=2,INST1B=3)
+      PARAMETER (INLAY=7,INTAU=6,IPCAA=158067)
+      PARAMETER (INST1=3,INST2=1,INST1B=2)
 C
 C ============================================================================
 C
@@ -96,8 +98,10 @@
 C                   matrices computed in MSMFT
 C ============================================================================
 C
-      PARAMETER (INT0=10,IPCUT=45,JSMAX=4)
-      PARAMETER (NLMB2=10000,NROM=380)
+C      PARAMETER (INT0=10,IPCUT=45,JSMAX=4)
+C      PARAMETER (NLMB2=10000,NROM=380)
+      PARAMETER (INT0=5,IPCUT=45,JSMAX=4)
+      PARAMETER (NLMB2=13223,NROM=81)
 C
 C ============================================================================
 C
@@ -116,9 +120,12 @@
 C  INLTOT    is the total number of sublayers in all CL's
 C  INLIN     Total number of inequivalent subplanes in all composite layers
 C
-      PARAMETER (INLTOT=24,NROMR=220,NROM2R=120)
-      PARAMETER (INLIN=24)
-      PARAMETER (JLMNI=392,JLMNI2=660)
+      ! PARAMETER (INLTOT=24,NROMR=220,NROM2R=120)
+      ! PARAMETER (INLIN=24)
+      ! PARAMETER (JLMNI=392,JLMNI2=660)
+      PARAMETER (INLTOT=9,NROMR=81,NROM2R=15)
+      PARAMETER (INLIN=9)
+      PARAMETER (JLMNI=72,JLMNI2=96)
 c     PARAMETER (JLMNI=160,JLMNI2=160)
 C
 C ============================================================================
diff -uN leedsatl.orig/satl2.f leedsatl/satl2.f
--- leedsatl.orig/satl2.f	2021-07-13 12:15:04.000000000 +0900
+++ leedsatl/satl2.f	2021-07-13 12:13:28.000000000 +0900
@@ -65,7 +65,7 @@
 C
 C ============================================================================
 C
-      PARAMETER (IPLMAX=8)
+      PARAMETER (IPLMAX=12)
 C
 C ============================================================================
 C
@@ -86,8 +86,8 @@
 C
 C ============================================================================
 C
-      PARAMETER (INLAY=8,INLTOT=18,INTAU=2,INST1=5)
-      PARAMETER (INLIN=18)
+      PARAMETER (INLAY=7,INLTOT=10,INTAU=6,INST1=3)
+      PARAMETER (INLIN=9)
 C
 C ============================================================================
 C
@@ -107,8 +107,8 @@
 C
 C ============================================================================
 C
-      PARAMETER (INT0=10,IPCUT=45,IINERG=100)
-      PARAMETER (INBED=32,IEERG=1600,JSMAX=4)
+      PARAMETER (INT0=5,IPCUT=45,IINERG=100)
+      PARAMETER (INBED=60,IEERG=2000,JSMAX=4)
 C
 C ============================================================================
 C
@@ -141,7 +141,7 @@
       DIMENSION ARB2(2),RBR1(2),RBR2(2),ACOORD(12,INLTOT,3)
       DIMENSION V(JPNL,2),JJS(JPNL,6)
       DIMENSION NTAUAW(INST1),MICUT(IPCUT),MJCUT(IPCUT)
-      DIMENSION ES(90),PHSS(90,80),PHSS2(90,80),ETH(INT0,IEERG)
+      DIMENSION ES(90),PHSS(100,100),PHSS2(100,100),ETH(INT0,IEERG)
       DIMENSION FPOS(INLAY,3),VPOS(INST1,INLAY,3)
       DIMENSION LAFLAG(INST1),LLFLAG(INLTOT)
       DIMENSION LAFLAG2(INLTOT,INST1),LPOINT(INLIN),LPBD(INLAY,INST1)
